devex/internal/pkg/rollout/rollout.go

```
@@ -57,65 +58,65 @@ func RevertToOriginalMCOImage(cs *framework.ClientSet, forceRestart bool) error
 
-	if err := setDeploymentReplicas(cs, cvoName, cvoNamespace, 1); err != nil {
+	if err := setDeploymentReplicas(ctx, cs, cvoName, cvoNamespace, 1); err != nil {
 		return fmt.Errorf("could not restore cluster version operator to default replica count of 1")
 	}
 
 	return nil
 }
 
-func ReplaceMCOImage(cs *framework.ClientSet, pullspec string, forceRestart bool) error {
-	if err := setDeploymentReplicas(cs, cvoName, cvoNamespace, 0); err != nil {
+func ReplaceMCOImage(ctx context.Context, cs *framework.ClientSet, pullspec string, forceRestart bool) error {
```

golangci-lint (2.12.2)

[warning] exported: exported function ReplaceMCOImage should have comment or be unexported

(revive)